repos/system_upgrade/el7toel8/actors/initramdiskgenerator/libraries/initramgen.py

```diff
@@ -65,16 +66,25 @@ def _install_files(context):
     in the initrd explicitly, so return the list of those files for other
     purposes.
     """
-    # TODO: currently we need this just for /etc/dasd.conf, but it's expected
+    # TODO: currently we need this just for two files, but it's expected
     # we will need this for more files in future. The concept used here will
     # need to be changed, as we should consume specific messages instead.
     # But for now this should be enough. Keeping that for future.
+    files = []
+
     if architecture.matches_architecture(architecture.ARCH_S390X):
         # we don't need to check existence of the file - it is required on this
         # this architecture
         context.copy_to('/etc/dasd.conf', '/etc/dasd.conf')
-        return ['/etc/dasd.conf']
-    return []
+        files.append('/etc/dasd.conf')
+
+    if next(api.consume(FstabSignal), None):
+        with open('/etc/fstab') as fstab7:
+            with context.open('/etc/fstab', 'w') as fstab8:
+                fstab8.writelines(drop_xfs_options(fstab7.readlines()))
+        files.append('/etc/fstab')
```

Comment:
what if user / vendor .. will need to change the fstab file as well? thinking abou that, this is not a good solution from my pov. and what is the content of the /etc/fstab inside the container by default? is it same as the original one already? if bydefault it is, the whole operation should happen just above this one file. as well, I guess separate function could be better - this file is always present inside the initrd and only thing here is that we want the upgradet versions inside the container. any else ideas welcomed @oamg/developers